ci/ubuntu.dockerfile

```diff
@@ -7,20 +7,23 @@ ARG venv=venv
 
 RUN echo "To invalidate cache"
 
+
 RUN apt-get update -y && apt-get install -y \
     python3-nacl \
-    libindy-crypto=0.4.5 \
     libindy=1.13.0~1420 \
+    libssl-dev \
```

Comment:
Do we need `libssl-dev` dependency there?

Reply:
yes. please see https://github.com/mac-arrap/ursa/blob/e165ddc9d160a4576fd30a28c6f610a858b1a3ec/docker/ubuntu/ursa-xenial.dockerfile#L12